framework/src/test/java/com/epam/ta/page/TenMinuteMailHomePage.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.ta.page;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class TenMinuteMailHomePage extends AbstractPage {
+
+    @FindBy(id = "mail_address")
+    WebElement mailAddress;
+
+    @FindBy(className = "small_subject")
+    WebElement openingMailButton;
+
+    @FindBy(xpath = "//tr[@id='mobilepadding']//td[2]/h3")
+    WebElement totalEstimatedCostFromMail;
+
+    public TenMinuteMailHomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    public TenMinuteMailHomePage openTenMinuteMailHomePage() {
+        ((JavascriptExecutor) driver).executeScript("window.open('https://10minutemail.com/')");
```

Comment:
why do you use JavascriptExecutor in this place?

Reply:
Click () method does not work in this place.